compiler/damlc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -244,3 +250,13 @@ data TypeRep = TypeRep Opaque
 instance Eq TypeRep where
   (==) = primitive @"BEEqual"
 #endif
+
+#ifdef DAML_UNSTABLE
```

Comment:
Please use a separate feature flag (e.g. `DAML_LAZY`) to gate this feature.

`DAML_UNSTABLE` is only for the unstable text functions, not a general flag for unstable features.

Reply:
Yep, I'll move it under a separate flag if we decide to merge it.

Reply:
Maybe worth renaming it to `DAML_UNSTABLE_TEXT`?